addons/website/static/src/js/menu/translate.js

```diff
@@ -310,13 +310,6 @@ var TranslatePageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
             var trans = self._getTranlationObject(this);
             trans.value = (trans.value ? trans.value : $node.html()).replace(/[ \t\n\r]+/, ' ');
         });
-        this._getEditableArea().prependEvent('click.translator', function (ev) {
-            if (ev.ctrlKey || !$(ev.target).is(':o_editable')) {
-                return;
-            }
-            ev.preventDefault();
-            ev.stopPropagation();
-        });
```

Comment:
Found strange this was entirely removed as it must have had an usecase... but I tracked it down to at least https://github.com/odoo/odoo/commit/8c559044a5e1be111dcb577ad7e5ab462c1002a0, we can safely say it is indeed probably useless